From fe3756ac75e4c27a1085c51662a017f75e3eccd7 Mon Sep 17 00:00:00 2001
From: Aster Boese <asterboese@mailbox.org>
Date: Tue, 19 Aug 2025 15:13:55 -0400
Subject: [PATCH] manually link libintl

Discussed with upstream and was decided to carry a patch in aports.

---
 src/app/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/app/CMakeLists.txt b/src/app/CMakeLists.txt
index 50e5853a..1832791b 100644
--- a/src/app/CMakeLists.txt
+++ b/src/app/CMakeLists.txt
@@ -16,7 +16,7 @@ add_executable(lomiri-filemanager-app ${filemanager_SRCS})
 target_include_directories(lomiri-filemanager-app PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
 
 target_link_libraries(lomiri-filemanager-app Qt5::Gui Qt5::Qml Qt5::Quick)
-target_link_libraries(lomiri-filemanager-app stdc++)
+target_link_libraries(lomiri-filemanager-app stdc++ -lintl)
 
 if(NOT "${CMAKE_CURRENT_SOURCE_DIR}" STREQUAL "${CMAKE_CURRENT_BINARY_DIR}")
 add_custom_target(lomiri-filemanager-app-qmlfiles ALL
-- 
2.50.1

